### PR TITLE
Amentum

### PR DESCRIPTION
**Pull request by indoptogopt**

### CODE REVIEW

File: gui/encyclopedia/articles/parent_articles/amentum.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "title": "The Amentum (Ankyle)",
+    "subtitle": "The ace of ancient javelineers",
+    "content": "The amentum (or “ankyle” in Greek) was a tool known to the ancient Greeks and Romans used to enhance the performance of a javelin throw. It was applied for sports, hunting, as well as warfare.\n\nThe amentum consisted of a short leather strap tied to the javelin's butt end. Throwers wound the amentum around the shaft a few times before gripping into the remaining loop with two fingers. When throwing, it essentially functioned as a lever, extending the arms reach and thus the distance the javelin would travel. Recreations estimate the gain at above 25% in speed and 50% in range. Moreover, the amentum also imparted spin on the javelin, improving its flight stability and accuracy.\n\nIn the final stages of the throwing motion, the amentum slipped off the hand and stayed attached to the javelin - a key difference to similarly functioning devices like the spear-thrower. Soldiers who used them likely brought the javelins to the battlefield already prepared."
```
Comment (TheShadowOfHassen) [inline gui/encyclopedia/articles/parent_articles/amentum.json]:
```suggestion
    "content": "The amentum (or “ankyle” in Greek) was a tool known to the ancient Greeks and Romans used to enhance the performance of a javelin throw. It was applied for sports, hunting, as well as warfare.\n\nThe amentum consisted of a short leather strap tied to the javelin's butt-end. Throwers wound the amentum around the shaft a few times before gripping into the remaining loop with two fingers. When throwing, it essentially functioned as a lever, extending the arm's reach and also the javelin's range. Recreations estimate the gain at above 25% in speed and 50% in range. Moreover, the amentum also imparted spin on the javelin, improving its flight stability and accuracy.\n\nIn the final stages of the throwing motion, the amentum slipped off the hand and stayed attached to the javelin - a key difference to similarly functioning devices like the spear-thrower. Soldiers who used them likely brought the javelins to the battlefield already prepared."

**Review by TheShadowOfHassen — COMMENTED** [review]
a few little changes. Spelling and a rewording. Looks good.

File: gui/encyclopedia/articles/parent_articles/amentum.json (hunk 1 above)
Comment (indoptogopt) [reply gui/encyclopedia/articles/parent_articles/amentum.json]: How about "extending the arm's reach and **therefore** the javelin's range"?

Comment (TheShadowOfHassen) [reply gui/encyclopedia/articles/parent_articles/amentum.json]: That works

**Comment by indoptogopt** [comment]
Can I merge it?

**Comment by TheShadowOfHassen** [comment]
Yep

**Pull request merged**